src/maintainer/adding_pkgs.rst

```diff
@@ -3,15 +3,19 @@
 Contributing packages
 *********************
 
+Process
+===================
```

Comment:
You should make the length of any header underlines be the same length as the text. Not a horrible thing, but it does cause warnings in the build log that can take up a lot of space and make it difficult to spot other problems.